### ISSUE TITLE
Add gitui

### ISSUE THREAD

**Issue by WaylonWalker** [webinstall/webi-installers#235]
https://github.com/extrawurst/gitui

gitui is a blazing fast gitui for the terminal. If you are often typing out git commands at the command line give gitui a try,  It takes fewer keystrokes (unless you have a crazy number of dirty files) and you can automatically see the diff of what you are committing in a split pane.

**Comment by WaylonWalker** [comment]
Do you have a good example to go from that uses binaries zipped in a github release?

**Comment by coolaj86** [comment]
@WaylonWalker Yes. Almost everything is that way. Take a look at any of the other rust tools: `rg`, `lsd`, `sd`, `watchexec`, etc.
